src/styles/components/_chart_d3.scss

```diff
@@ -0,0 +1,94 @@
+.chart-d3 {
+  position: relative;
+  height: 350px;
```

Comment:
The majority of this styling was moved from score-barchart.scss, so that the styles could be shared with median-barchart. I'll comment on the styling that isn't pulled from score-barchart